gemato/openpgp.py

```diff
@@ -383,7 +385,15 @@ def home(self):
         assert self._home is not None
         return self._home
 
+    @property
+    def env(self):
+        return self._env
+
+    def update_env(self, key, value):
+        if key in ENVIRON_WHITELIST:
+            self._env[key] = value
```

Comment:
I'm not sure to get it but here it's only used for GNUPGHOME but in case of providing values from Portage configuration, not only: https://github.com/gentoo/portage/pull/597/files#diff-763369bc335cf0fd433abc8d81028823R259

Reply:
Values from Portage configuration should be reexported to `os.environ`.

Reply:
So maybe this part you could have a look at current Portage implementation and give me your feedback of what would be the best solution for that instead of using the current code made on the PR please?

Reply:
Let's focus on fixing the gemato standalone first, then look what could be used inside Portage and maybe change gemato to handle that.